README.md

```diff
@@ -13,83 +13,122 @@ Create baseline images and test for CSS regression during standard Ember tests u
+
+    - install `ember-cli-visual-acceptance`
+    - run `ember test -s` (this will launch a browser)
+    - open the console and type `window.ui`
+
+
+To target specific versions, you can edit `ember-cli-build.js` as follows:
+
 ```javascript
 visualAcceptanceOptions: {
-      imageDirectory: 'visual-acceptance'
-    }
-``` 
-* You can specify browsers to target by adding an object inside the `targetBrowsers` array in the `visualAcceptanceOptions` of `ember-cli-build.js`
+  targetBrowsers: [{
+    browser: "Chrome",
+    os: "Mac OS X",
+    osversion: "10.11.2",
+    version: "49.0.2623.112"
+  }]
+}
+```
+
+Also, rather then specifying exact version for the browser and the OS versions ,you can append `>=` as follows:
 ```javascript
 visualAcceptanceOptions: {
-      targetBrowsers: [{
-        browser: "Chrome",
-        os: "Mac OS X",
-        osversion: "10.11.2",
-        version: "49.0.2623.112"
-      }]
-    }
+  targetBrowsers: [{
+    browser: "Chrome",
+    os: "Mac OS X",
+    osversion: ">=10.11.2",
+    version: ">=49.0.2623.112"
+  }]
+}
 ```
-  * The browser object is comes from [detect.js](https://github.com/benbscholz/detect/blob/master/src/detect.js#L6-L11). You can view your current browsers object after installing `ember-cli-visual-acceptance` by visiting typing in `window.ui` into the Browser's console after running `ember test -s`
-  * You can append `>=` to the begininng of the `osversion` and `version` if you also wish to target versions that are greater than or equal to the targeted version
-  * You can also exclude both `osversion` and `version` if only the browser and OS matters to you
+You can also omit the `osversion` and `version` if not needed.
+
+## API
+
+```javascript
+capture(imageName, height, width, misMatch, imageDirectory)
+```
+
+
+|           Name           | Type   | Default             | Description                           |
+|--------------------------|--------|---------------------|---------------------------------------|
+| imageName                | string | required            | Name of the image you wish to save    |
+| height                   | number | null                | Define the height of the canvas in pixels. If null, renders with full height of the window. |
+| width                    | number | null                | Define the width of the canvas in pixels. If null, renders with full width of the window.   |
+| misMatch                 | float  | 1.00                | The maximum percentage ResembleJs is allowed to misMatch. |
+| imageDirectory           | string | 'visual-acceptance' | The location where the `-passed.png` and `-failed.png` images will be saved. Note: Cannot be within the `tests` folder as this will restart the test every time an image is saved |
```

Comment:
I need to double check if we still need this parameter. I'll add a reply tomorrow on this